File: Clg_Placement/Java/_32.BinarySortingArray.cpp
class Solution{
  public:
    
    // A[]: input array
    // N: input array
    //Function to sort the binary array.
    void binSort(int A[], int N)
    {
       int low=0,high=N-1;
       while(low<high){
           if(A[low]==0) low++;
           else if(A[low]==1) swap(A[low],A[high--]);
       }
    }
};

File: Clg_Placement/Java/_32.BinarySortingArray.java
class Solution
{
    //Function to sort the binary array.
    static void binSort(int arr[], int N)
    {
        int low=0;
        int high=N-1;
        while(low<high){
            if(arr[low]==0) low++;
            else if(arr[low]==1){
                arr[low]=arr[low]^arr[high];
                arr[high]=arr[low]^arr[high];
                arr[low]=arr[low]^arr[high];
                high--;
            }
        }
    }
}
